e2e/testvalues/values.go

```diff
@@ -72,7 +74,7 @@ var SimdNewGenesisCommandsFeatureReleases = semverutil.FeatureReleases{
 	MajorVersion: "v8",
 }
 
-// TransferSelfParamsFeatureReleases represents the releases the transfer module started managing its own params.
+// SelfParamsFeatureReleases represents the releases the transfer module started managing its own params.
```

Comment:
nice catch! Needs some additional tweaking:

```suggestion
// SelfParamsFeatureReleases represents the releases where the ibc modules started managing their own params.
```